package/MDAnalysis/analysis/rms.py

```diff
@@ -398,125 +416,100 @@ def __init__(self, traj, reference=None, select='all',
                         igroup, sel['reference'], sel['mobile'],
                         len(atoms['reference']), len(atoms['mobile'])))
 
+        # necessary for throwing error in save, let me know if there is a
+        # better way
```

Comment:
Initializing with `None` seems fine.

Remove the "let me know if there is a better way".